src/pages-and-resources/data/api.js

```diff
@@ -8,19 +8,14 @@ ensureConfig([
   'STUDIO_BASE_URL',
 ], 'Course Apps API service');
 
-const apiBaseUrl = getConfig().STUDIO_BASE_URL;
-
-const courseAppsApiUrl = `${apiBaseUrl}/api/course_apps/v1/apps`;
-const courseAdvancedSettingsApiUrl = `${apiBaseUrl}/api/contentstore/v0/advanced_settings`;
-
 /**
  * Fetches the course apps installed for provided course
  * @param {string} courseId
  * @returns {Promise<[{}]>}
  */
 export async function getCourseApps(courseId) {
   const { data } = await getAuthenticatedHttpClient()
-    .get(`${courseAppsApiUrl}/${courseId}`);
+    .get(`${getConfig().STUDIO_BASE_URL}/api/course_apps/v1/apps/${courseId}`);
```

Comment:
Can we make `apiBaseUrl` a function instead?  Like:

```
const getApiBaseUrl = () => getConfig().STUDIO_BASE_URL;
const getCourseAppsApiUrl = () => `${getApiBaseUrl()}/api/course_apps/v1/apps`;
...
```

Reply:
Good point, I think mostly it shall work, because it will not define the config the file/module is loaded rather when teh fucntion is call... I will test it and if it works as expected I will apply the changes